bloom/infra/marine_traffic_scraper.py

```diff
@@ -35,7 +35,7 @@ def __init__(
     def __enter__(self) -> Chrome:
         self._driver = Chrome(
             options=self._options,
-            version_main=109,
+            version_main=110,
```

Comment:
If we update Chrome version in the scrapper, we also have to update version of chrome installed in Docker otherwise we would face incompatible driver error when running app.py.

Reply:
Yes, that's why I dislike this library. We should be able not to specify chrome version & make it run from different setups

Reply:
Maybe the best practice is to define chrome version only in Docker file and then in this library detect with os which version is installed in the docker and adapt the value of version_main based on the version installed.

Reply:
could we use an environment variable ? It can be read from both Dockerfile and python, but we have to load it somehow ...

Reply:
fixed in b882bf295485474e505995e7105fb43b2318b079